dev/cli

```diff
@@ -2,4 +2,11 @@
 
 set -eu
 
+# Get the directory where the script is located
+SCRIPT_DIR=$(dirname "$(realpath "$0")")
+
+# Navigate to the top-level directory
+TOP_LEVEL_DIR=$(realpath "$SCRIPT_DIR/..")
+cd "$TOP_LEVEL_DIR"
```

Comment:
A nice-to-have would be to use `pushd` and `popd` to leave the user back where they started, but it's up to you

Reply:
AFAIU the `cd` only affects the scope of the script. It does not actually move the user to the new directory.

You can see it in the examples.